packages/pipeline/test/data_sources/relayer-registry/index_test.ts

```diff
@@ -0,0 +1,24 @@
+import * as chai from 'chai';
+import 'mocha';
+
+import { getRelayerInfoAsync, RelayerResponses } from '../../../src/data_sources/relayer-registry';
+import { chaiSetup } from '../../utils/chai_setup';
+
+const RELAYER_REGISTRY_URL =
```

Comment:
Hmm... this unit test requires an internet connection? Can we save the json as a fixture and mock the endpoint instead?

Reply:
➕ IMO we shouldn't send any requests to third party servers as part of our normal test suite. The reason to avoid this is that third party APIs can break (some of the specific ones we're using break more frequently than I would like), resulting in intermittent and confusing errors for other developers on the team.